rcl/src/rcl/event.c

```diff
@@ -190,6 +185,18 @@ rcl_event_get_rmw_handle(const rcl_event_t * event)
   }
 }
 
+bool
+rcl_event_is_valid(const rcl_event_t * event)
```

Comment:
It would be nice to also check the validity of `event->impl->rmw_handle.event_type`